gradle/libs.versions.toml

```diff
@@ -22,6 +22,7 @@ androidImageCropper = "4.5.0"
 apng = "2.25.0"
 awebp = "2.25.0"
 glideCompiler = "4.15.1"
+zjupureWebpdecoder = "2.3.4.15.1"
```

Comment:
why do we need a new library if animating gifs in files works already?

Reply:
Animating thumbnails of GIFs is working, but not for WEBPs. This library is a glide extension to extend support for WEBP files. This is required to get animated thumbnails for animated WEBP files working.